_data/jobs.yml

```diff
@@ -1,3 +1,8 @@
+- expires: 2022-07-27
+  location: Department of Data Science, Dana-Farber Cancer Institute, Boston, MA
+  name: Research Software Engineer
+  posted: 2022-06-28
+  url: https://careers-dfci.icims.com/jobs/29995/research-software-engineer%2c-data-science/job?mode=view&mobile=false&width=1440&height=500&bga=true&needsRedirect=false&jan1offset=-300&jun1offset=-240
```

Comment:
```suggestion
  url: https://careers-dfci.icims.com/jobs/29995/research-software-engineer%2c-data-science/job
```

Reply:
maybe a simpler URL that doesn't specify how to view the post?

Reply:
Good catch.  I simply included verbatim what the submitter put into the Google form.